packages/paste-design-tokens/gulpfile.ts

```diff
@@ -273,7 +307,9 @@ gulp.task(
     'tokens:android',
     'tokens:raw',
     'tokens:sketchpalette',
-    'tokens:gatsby'
+    'tokens:gatsby',
+    'tokens:gatsby:js',
+    'tokens:gatsby:d:ts'
```

Comment:
The original build is left untouched, the change was that two new builds were added.

To clarify, is the chore a blocking change for this PR or is this a more general note about the future?

Reply:
According to conventional comments https://conventionalcomments.org:

> Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.